### ISSUE TITLE
copy image to jupyter notebook can not effect!

### ISSUE THREAD

**Issue by jianboy** [jupyterlab/jupyterlab#7537]
## Description

I copy a image to jupyter notebook, it add it:
```
![image.png](attachment:image.png)
```
![2019-11-19_204151](https://user-images.githubusercontent.com/11532015/69147605-5e222d80-0b0d-11ea-9b03-e880dd971f00.png)

but can not preview to others.

## Reproduce

2. Click on '...'
3. Scroll down to '...'
4. See error '...'

## Context

- Operating System and version:
- Browser and version:
- JupyterLab version:

<details><summary>Troubleshoot Output</summary>
<pre>
Paste the output from running `jupyter troubleshoot` from the command line here.
You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Browser Output</summary>
<pre>
Paste the output from your browser Javascript console here.
</pre>
</details>

**Comment by agoose77** [comment]
Hi @jianboy,

Could you add some more information such as your JupyterLab version, and perhaps a GIF to reproduce the error?

**Comment by jianboy** [comment]
> Hi @jianboy,
> 
> Could you add some more information such as your JupyterLab version, and perhaps a GIF to reproduce the error?

Jupyterlab version: Version 1.2.3
![image](https://user-images.githubusercontent.com/11532015/69717831-c85b5380-1147-11ea-91fa-22783b01a6ee.png)

**when I copy a screenshot to jupyterlab, It can not auto add a picture to current folder.**

[Video_2019-11-27_190342.zip](https://github.com/jupyterlab/jupyterlab/files/3896529/Video_2019-11-27_190342.zip)

**Comment by agoose77** [comment]
I'm not sure if I understand what the issue is.
These images are added as *cell attachments*, which means they are stored inside the notebook as per-cell data. Consequently, they are not written to disk.

**Comment by jianboy** [comment]
> I'm not sure if I understand what the issue is.
> These images are added as _cell attachments_, which means they are stored inside the notebook as per-cell data. Consequently, they are not written to disk.

yes, I know, but saving picture to current folder which is a good idea. you can see other markdown editor,below:

![image](https://user-images.githubusercontent.com/11532015/69718846-3ef95080-114a-11ea-97f7-826b75d4489b.png)



it may not a bug, but it is a good feature.

**Comment by jianboy** [comment]
jupyter only create a file, so it can insert the picture as a byte format :

```
<img src="data:image/jpeg;base64,iVBxxxxxxxxxxxxxxxxxxxx...">
```
